docker_images/mbed-os-env/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# ------------------------------------------------------------------------------
+# Pull base image
+FROM ubuntu:20.04
+
+# ------------------------------------------------------------------------------
+# Arguments
+ARG WORKDIR=/root
+
+# ------------------------------------------------------------------------------
+# Install tools via apt
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -x \
+    && apt -y update \
+    && apt -y install \
+    git \
+    wget \
+    python3 \
+    python3-dev \
+    python3-setuptools \
+    python3-usb \
+    python3-pip \
+    software-properties-common \
+    build-essential \
+    astyle \
+    mercurial \
+    ninja-build \
+    libssl-dev \
+    cargo \
+    flex \
+    bison \
+    && apt clean && rm -rf /var/lib/apt/lists \
+    && update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1 \
+    && : # last line
+
+RUN set -x \
+    && pip3 install -U cmake
+
+WORKDIR /tmp/
+
+RUN set -x \
+    && git clone --depth=1 --single-branch --branch Release_1_8_14 https://github.com/doxygen/doxygen \
```

Comment:
Can't we install a good enough Doxygen with apt? Maybe we used to have to do this when the distribution we used didn't include a new enough Doxygen version, but [Ubuntu 20.04 provides Doxygen 1.8.17](https://packages.ubuntu.com/focal/doxygen) which should be new enough.

Reply:
will use apt to install. Thanks